### ISSUE TITLE
Night mode and visually impaired

### ISSUE THREAD

**Issue by hippybear** [FreeCodeCamp/FreeCodeCamp#9838]
The light grey labels in the instructions portion of the screen have neon green text color. That is not the best color scheme for people with visual issues. There is a chrome plugin and some script to change these things locally but that shouldn't really be something the user needs to do is it? I don't know. Maybe it isn't really an issue. If it isn't go on and close this I apologize.

**Comment by erictleung** [comment]
@hippybear thanks for the feedback. What color scheme do you suggest would work better? We strive to be as accommodating as possible.

cc/ @FreeCodeCamp/issue-moderators

**Comment by hippybear** [comment]
Personally if the text inside the label was a dark color it would work fine for me. Black or dark blue maybe?

**Comment by hallaathrad** [comment]
Hmmm You're right. this could be better. I'm on it.
